src/singletons/Settings.hpp

```diff
@@ -201,6 +201,11 @@ class Settings : public ABSettings, public ConcurrentSettings
     BoolSetting removeSpacesBetweenEmotes = {
         "/emotes/removeSpacesBetweenEmotes", false};
 
+    BoolSetting bttvGlobalEmotes = {"/emotes/bttvGlobal", true};
+    BoolSetting bttvChannelEmotes = {"/emotes/bttvChannel", true};
+    BoolSetting ffzGlobalEmotes = {"/emotes/ffzGlobal", true};
+    BoolSetting ffzChannelEmotes = {"/emotes/ffzChannel", true};
```

Comment:
Store the settings one level further down, e.g. `/emotes/bttv/global` and `/emotes/ffz/channel`
I would also consider adding some verbiage in at least the setting variable like `load...` or `...Enabled` or `enable...`